configs/components/rubygem-highline.rb

```diff
@@ -7,6 +7,8 @@
     pkg.md5sum '4209083bda845d47dcc05b7ab23f25fd'
   when '3.0.1'
     pkg.sha256sum 'ca18b218fd581b1fae832f89bfeaf2b34d3a93429c44fd4411042ffce286f009'
+  when '3.1.0'
+    pkg.sha256sum '14183e4628f41fb4211766d0d38fb153f01f1ec39adfa83f5f7c6e6ebeb2b8eb'
```

Comment:
Same comment, throughout all these case statements, i'm not seeing how these are used. Perhaps you missed logic in project files? Are all of these due to ruby3/puppet8?

Reply:
Removed the case for 3.0.1 as 3.1.0 is the newest for Ruby 3

Reply:
Removed 3.1.0 case